annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/field/FieldDeclarationInfo.java

```diff
@@ -1,14 +1,13 @@
 package edu.ucr.cs.riple.core.metadata.field;
 
 import com.google.common.collect.ImmutableSet;
-import edu.ucr.cs.riple.core.metadata.Hashable;
 import java.nio.file.Path;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
 /** Used to store information regarding multiple field declaration statements in classes. */
-public class FieldDeclarationInfo implements Hashable {
```

Comment:
For nonnull store `NonnullStore extends MetaData<Location>`, `Location` instance requires to implement `Hashable`. But `Hashable` is declared in annotator-core and `Location` is declared in injector. Since core depends on Injector, the only way to resolve this was to declare `Hashable` in injector, which does not make any sense since it will not be used anywhere in injector. I wanted to have this interface as a marker, but I don't think it even worths having this anymore. In the early stages, we had an internal data structure similar to `Multimap`, but after I found about `Multimap` I replaced it with `MultiMap` and this interface is redundant. I never had a chance to remove it until this PR that it is actually causing a problem. Please let me know if you would like me to remove it in a separate PR. Thank you.

Reply:
No this is fine, no need to separate, thanks